Python_DataEngineering/Database.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import print_function
+
+import mysql.connector
+from mysql.connector import errorcode
+
+DB_NAME = 'BikePoints'
+
+TABLES = {}
+TABLES['bikes_availablility'] = (
+    "CREATE TABLE `bikes_availability` ("
+    "  `bikeid` int(11) NOT NULL,"
+    "  `nam` varchar(50) NOT NULL,"
+    "  `lat` varchar(50) NOT NULL,"
+    "  `longitude` varchar(50) NOT NULL,"
+    "  `installed` enum('true','false') NOT NULL,"
+    "  `locked` enum('true','false') NOT NULL,"
+    "   `nbBikes` int(3),"
+    "   `nbEmptyDocks`  int(3),"
+    "  PRIMARY KEY (`bikeid`)"
+    ")")
+
+TABLES['timestamp_bikes_data'] = (
+    "CREATE TABLE `timestamp_bikes_data` ("
+    "  `bikeid` int(11) NOT NULL,"
+    "  `nam` varchar(50) NOT NULL,"
+    "  `lat` varchar(50) NOT NULL,"
+    "  `longitude` varchar(50) NOT NULL,"
+    "  `installed` enum('true','false') NOT NULL,"
+    "  `locked` enum('true','false') NOT NULL,"
+    "   `TimeSnap` datetime,"
+    "   `nbBikes` int(3),"
+    "   `nbEmptyDocks`  int(3),"
+    "  PRIMARY KEY (`TimeSnap`)"
+    ")")
+
+
+cnx = mysql.connector.connect(user='root',password='1234')
```

Comment:
DB Secrets should be stored in environment variables for security, rather than in code

Reply:
I added the variables in environment and used those variables. Thanks for it.